image-trigger

```diff
@@ -44,6 +44,7 @@ REFRESH = {
     "fedora-coreos": {},
     "fedora-rawhide": {},
     "fedora-rawhide-boot": {},
+    "fedora-rawhide-anaconda-payload": {"refresh-days": 30},
```

Comment:
I assume this can be triggered manually at from this, right?

Reply:
Similarly to triggering image refreshes- this can be triggered by creating an issue like this.
https://github.com/cockpit-project/bots/issues/2947

Reply:
Yes, or on the CLI with `./image-trigger fedora-rawhide-anaconda-payload` (which creates such a GitHub issue via the API)